catroid/src/main/java/org/catrobat/catroid/content/actions/ChangeTempoByAction.kt

```diff
@@ -20,42 +20,36 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.catrobat.catroid.content.actions;
-
-import android.util.Log;
-
-import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
-
-import org.catrobat.catroid.content.Scope;
-import org.catrobat.catroid.formulaeditor.Formula;
-import org.catrobat.catroid.formulaeditor.InterpretationException;
-import org.catrobat.catroid.pocketmusic.mididriver.MidiSoundManager;
-
-public class ChangeTempoByAction extends TemporalAction {
-
-	private Scope scope;
-	private Formula tempo;
-
-	@Override
-	protected void update(float delta) {
-		try {
-			int newTempo;
-			if (tempo == null) {
-				newTempo = Integer.valueOf(0);
-			} else {
-				newTempo = tempo.interpretInteger(scope);
-			}
-			MidiSoundManager.getInstance().setTempo(MidiSoundManager.getInstance().getTempo() + newTempo);
-		} catch (InterpretationException interpretationException) {
-			Log.d(getClass().getSimpleName(), "Formula interpretation for this specific Brick failed.", interpretationException);
-		}
-	}
-
-	public void setTempo(Formula tempo) {
-		this.tempo = tempo;
-	}
-
-	public void setScope(Scope scope) {
-		this.scope = scope;
-	}
+package org.catrobat.catroid.content.actions
+
+import android.util.Log
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction
+import org.catrobat.catroid.content.Scope
+import org.catrobat.catroid.formulaeditor.Formula
+import org.catrobat.catroid.formulaeditor.InterpretationException
+import org.catrobat.catroid.pocketmusic.mididriver.MidiSoundManager
+
+class ChangeTempoByAction : TemporalAction() {
+    private var scope: Scope? = null
```

Comment:
The private keyword is not needed here, kotlin assumes it to be automatically private. Consider changing these two lines to:
lateinit var scope: Scope
lateinit var tempo: Formula
This way the variables do not have to be set nullable.